src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -149,14 +149,22 @@ public function text($message)
     {
         $this->autoPrependText();
 
-        if (!is_array($message)) {
-            $this->writeln(sprintf(' // %s', $message));
-
-            return;
+        $messages = is_array($message) ? array_values($message) : array($message);
+        foreach ($messages as $message) {
+             $this->writeln(sprintf(' %s', $message));
         }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function comment($message)
```

Comment:
I am +1 for removing the interface as well. There was a reason for it that was discussed during the initial PR but I can't find it. Laravel is using this feature but they extend `SymfonyStyle`.